### PR TITLE
Bugs

### PR DESCRIPTION
**Pull request by Drumor**
These are some changes ask by our accountant

### CODE REVIEW

File: ingifet.py [hunk 1]
```diff
@@ -74,10 +74,11 @@ def POST(self, id=None):
...
             return render.edit_user(form, user,"")
-
-        for u in User.all():
-            if form.d.email == u.email:
-                return render.edit_user(form, user, 'NotUniqueEmail')
+        #check only if email changed
+        if user.email != form.d.email:
+            for u in User.all():
+                if form.d.email == u.email:
+                    return render.edit_user(form, user, 'NotUniqueEmail')
```
Comment (anthonygego) [inline ingifet.py]: This is not directly related to the patch, but wouldn't it be more efficient to query a User based on the form email instead of looping on all the user list ?

Comment (Drumor) [reply ingifet.py]: I tought the same thing after making the pull request. I'll do the modification.

Comment (Drumor) [reply ingifet.py]: Resolved by 8e87912
